### PR TITLE
11 - Adds installation manual to setup Judo locally

### PR DESCRIPTION
**Pull request by athityakumar**
- [x] Does all of your new or changed code have unit tests?
- [x] Is overall unit test statement coverage still above 85%?
- [x] Is code style linting passing with `npm run lint`?
- [x] Is the related GitHub issue number included in your branch name? ex branch name: `fix/12-some-bug`
- [x] Is the related GitHub issue number included in your pull request title? example PR title: "12 - Fixes some bug"
- [x] Is documentation still up to date after your change?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +47,28 @@ npm i -g @intuit/judo
 judo <tests-dir>
 ```
 
+OR if you would like to locally develop Judo:
```
Comment (ejfrancis) [inline README.md]: Could you make this its own section w/a heading? So change it to this:

```
## Building for Local Development

If you would like to locally develop Judo:
```

File: README.md [hunk 2]
```diff
@@ -45,6 +47,28 @@ npm i -g @intuit/judo
...
+git clone https://github.com/intuit/judo
+
+# go to the judo directory
+cd judo
+
+# then install the dependencies
+npm i
+
+# next build the src/ files into dist/
+npm run
```
Comment (ejfrancis) [inline README.md]: this should be `npm run build`

**Comment by ejfrancis** [comment]
hey @hipstersmoothie it looks like the circleci config uses auto to tag a release on github, but it doesn't actually run `npm publish` or `yarn publish` to publish it to the registry so I'm unable to get this released as a patch. could you add that to the config so it's automated?

**Comment by hipstersmoothie** [comment]
I'm not sure what you mean. It looks like v0.0.15 went out with these changes https://www.npmjs.com/package/@intuit/judo

**Comment by ejfrancis** [comment]
ah great, must have just taken awhile for cache to update then 👍  i had checked that page about 24 hours after the job and still didn't see it

**Pull request merged**